.github/workflows/test.yml

```diff
@@ -102,3 +102,17 @@ jobs:
           cmd: npx nx run storybook:visual-regression-branch
           affectedCmd: npx nx affected --target=visual-regression-pr
           isDev: ${{ env.IS_DEV }}
+  test-build:
```

Comment:
It's a suggestion, as long as for now we are not checking that build is working.